pkg/registry/core/componentstatus/validator.go

```diff
@@ -72,7 +72,11 @@ func (server *Server) DoServerCheck() (probe.Result, string, error) {
 	}
 	url := utilnet.FormatURL(scheme, server.Addr, server.Port, server.Path)
 
-	result, data, err := server.Prober.Probe(url, nil, probeTimeOut)
+	req, err := httpprober.NewProbeRequest(url, nil)
+	if err != nil {
+		return probe.Unknown, "", err
```

Comment:
nit: add some error context
```suggestion
		return probe.Unknown, "", fmt.Errorf("failed to construct probe request: %w", err)
```